public/catch-js-errors.js

```diff
@@ -89,23 +92,32 @@
 	}
 
 	if ( isLocalStorageNameSupported() ) {
+		assignment = Math.random();
 		// Randomly assign 1% of users to log errors
-		if ( localStorage.getItem( 'log-errors' ) === undefined ) {
-			if ( Math.random() <= 0.01 ) {
-				localStorage.setItem( 'log-errors', 'true' );
+		if ( ! localStorage.getItem( 'log-errors' ) ) {
+			if ( assignment <= 0.01 ) {
+				localStorage.setItem( 'log-errors', 'atlas' );
```

Comment:
I don't want to get hung up on names, but this would be better as `rest-api`. `atlas` is the javascript-based system that we used to run, not the mechanism for tracking we are making use of here. What we're doing here has nothing to do with Atlas, other than that we're following a similar approach.